### PR TITLE
fix: creating new item in basket instead of updating quantity

### PR DESCRIPTION
**Pull request by N-Feist**
[[AB#104440](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/104440)]

## Why:

Closes: #ID

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

**Review by FabianGerke — DISMISSED** [review]

File: apps/web/components/ui/PurchaseCard/PurchaseCard.vue [hunk 1]
```diff
@@ -196,10 +196,11 @@ const handleAddToCart = async () => {
     return;
   }
 
+  const orderProperties = getPropertiesForCart();
   const params = {
     productId: Number(productGetters.getId(product.value)),
     quantity: Number(quantitySelectorValue.value),
-    basketItemOrderParams: getPropertiesForCart(),
+    basketItemOrderParams: orderProperties.length > 0 ? orderProperties : undefined,
```
Comment (maxiroellplenty) [inline apps/web/components/ui/PurchaseCard/PurchaseCard.vue]: can't we rewrite the getOrderProperties method to return undefined if there are no properties? Or am i missing something?

**Review by maxiroellplenty — DISMISSED** [review]

File: apps/web/components/ui/PurchaseCard/PurchaseCard.vue (hunk 1 above)
Comment (FabianGerke) [reply apps/web/components/ui/PurchaseCard/PurchaseCard.vue]: Then we need to update "getPropertiesPrice" too.

Comment (N-Feist) [reply apps/web/components/ui/PurchaseCard/PurchaseCard.vue]: already did in the changes from this morning. Just didn't pushed yet. But not sure, what to do with "getPropertiesPrice", when properties = undefined. Right now im just returning 0.

File: apps/web/composables/useProductOrderProperties/useProductOrderProperties.ts [hunk 2]
```diff
@@ -116,18 +117,21 @@ export const useProductOrderProperties: UseProductOrderPropertiesReturn = () =>
...
+          const labels = productPropertyGetters.getOrderPropertyLabel(propertyItem);
+          if (labels.surchargeType === 'incl') {
+            price += productPropertyGetters.getOrderPropertySurcharge(propertyItem);
+          }
         }
-      }
-    });
-    return price;
+      });
+      return price;
```
Comment (FabianGerke) [inline apps/web/composables/useProductOrderProperties/useProductOrderProperties.ts]: Remove line 132

File: apps/web/composables/useProductOrderProperties/useProductOrderProperties.ts [hunk 3]
```diff
@@ -116,18 +117,21 @@ export const useProductOrderProperties: UseProductOrderPropertiesReturn = () =>
...
+    }
+    return 0;
```
Comment (FabianGerke) [inline apps/web/composables/useProductOrderProperties/useProductOrderProperties.ts]: line 134: return price;

**Review by FabianGerke — CHANGES_REQUESTED** [review]

**Review by FabianGerke — APPROVED** [review]

**Review by FabianGerke — APPROVED** [review]

**Pull request merged**